### DISCUSSION TITLE
Gathering improvement ideas

### DISCUSSION THREAD

**Discussion by rlouf** [blackjax-devs/blackjax#4] [New features]
I copy in this thread all the improvement suggestions that were made here and there.

1. In the integrators we use `jax.grad` to differentiate the kinetic energy. However we can compute a closed-form derivative which will be faster to evaluate, which could implement it in the `gaussian_euclidean_metrics`. Since the gradient of the kinetic energy is likely a lot faster to evaluate that the potential we don't expect a noticeable improvement for large models.
